builtin/game/chat.lua

```diff
@@ -923,7 +923,7 @@ core.register_chatcommand("time", {
 		end
 		core.set_timeofday((hour * 60 + minute) / 1440)
 		core.log("action", ("%s sets time to %d:%02d"):format(name, hour, minute))
-		return true, "Time of day changed."
+		return true, ("Time of day changed to %d:%02d."):format(hour, minute)
```

Comment:
A few lines before, there is another occurrence of this same message (issued when using old syntax like `/time 6000`), it could be changed the same way.